### PR TITLE
Added freecodecamp.org link

### PR DESCRIPTION
**Pull request by theblobscp**
**Please describe the changes this PR makes and why it should be merged:**
I learned pretty much everything from freecodecamp.org, I wanted to let others know about it. It adds a link to freecodecamp.org, and should be added due to how popular it is for learning how to code

### CODE REVIEW

**Comment by J-Human** [comment]
freeCodeCamp would be a good site to learn web development, so it would be good if it was added. But, I think the text that is shown in the hyperlink should be more specific, in terms of how freeCodeCamp would help with JavaScript itself (since the motive of the first page is to tell people to learn so, if they need to).

**Comment by theblobscp** [comment]
Anything in particluar? I kinda suck at naming things

File: guide/README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ If you don't know JavaScript but would like to learn about it, here are a few li
 * [Nodeschool, for both JavaScript and Node.js lessons](https://nodeschool.io/)
 * [MDN's JavaScript guide and full documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
 * [Google, your best friend](https://google.com)
+* [Freecodecamp, where you can learn other stuff as well](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/)
```
Comment (J-Human) [inline guide/README.md:26]:
```suggestion
* [Freecodecamp, a popular site to learn web development, including JavaScript](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/)
```

**Review by J-Human — CHANGES_REQUESTED** [review]

**Comment by theblobscp** [comment]
I added your changes

**Comment by Danktuary** [comment]
Sorry for the delayed response, but I don't think we need any more resources listed than what we already have. 😅

**Pull request closed without merging**